notebooker/web/templates/results.html

```diff
@@ -21,6 +21,15 @@ <h2>{{ result.report_title }}</h2>
                         class="iframeToLoad"
                         scrolling="no">
                 </iframe>
+                <div class="ui modal" style="width: 50%; height: 70%;" id="stdoutModal">
+                  <div class="header">Stdout</div>
+                  <div class="content">
+                    <textarea readonly style="white-space: pre-line; width: 100%; height: 100%;" id="stdoutContent"></textarea>
```

Comment:
Is there a way to wrap this in `<code>` or something within fomantic-ui which will show this as monospace?